main.go

```diff
@@ -289,6 +311,11 @@ func main() {
 		log.RInfof(stepID, "cache_archive_size", data, "Size of extracted cache archive: %d Bytes", cacheRecorderReader.BytesRead)
 	}
 
+	err = writeCachePullTimestamp()
```

Comment:
it is a good approach to keep error var scope as short as possible.
I suggest moving this to the if statement:

```
if err := writeCachePullTimestamp(); err != nil {
```

Reply:
might worth to write this file in each successful return of the step, like here: https://github.com/bitrise-steplib/steps-cache-pull/pull/25/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R271